src/PrestaShopBundle/Resources/config/routing.yml

```diff
@@ -6,3 +6,8 @@ _admin_routing:
 _api_routing:
   resource: "@PrestaShopBundle/Resources/config/routing/api.yml"
   prefix: /api
+
+_new_api_routing:
+  resource: .
```

Comment:
What "." does it mean for resource ?

Reply:
From what I understood, for api_platform type of routing, this resource entry is mandatory but not used, so only putting a dot does the job (That's the way it's done in api platform itself: https://github.com/api-platform/api-platform/blob/main/api/config/routes/api_platform.yaml).

Reply:
Every RouteLoader (xml, yaml, annotation) needs a path,  and api platform does not need it

Reply:
ok thank you :smile: